src/containers/App/App.js

```diff
@@ -3,30 +3,37 @@ import { func } from 'prop-types';
 import { translate } from 'react-i18next';
 import 'i18n/i18n';
 import styles from './assets/app.scss';
+import { initializeGA, trackCurrentPage } from '../../googleAnalytics';
 
-export function App({ t }) {
-  return (
-    <div className={styles.app}>
-      <img src={require('./assets/mentor.jpg')} alt="Mentor" className={styles.image} />
-      <header className={styles.header}>
-        <h1 className={styles.title}>{t('landing-page-header')}</h1>
-        <h4 className={styles.tagline}>{t('landing-page-subheader')}</h4>
-        {/* TODO: create a React Component for a `Button` and add scoped styles */}
-        <a
-          className="button button__white--fill button__link"
-          href="https://docs.google.com/forms/d/1TraoygcJay39cLiBQM-ROlToy7xIgcdx_VvCQuE1lJw/edit#responses"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          {t('landing-page-start')}
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  componentDidMount() {
+    initializeGA();
+    trackCurrentPage();
```

Comment:
Both These calls can be part of a separate file like analytics.js..
suppose tomorrow we want to include mixpanel, or heap or some other lib, we add that in the abstracted analytics.js and it gets applied everywhere where ever analytics.js has been invoked

Reply:
You've suggested to abstract the Google Analytics code. I thought that was a good idea so I abstracted it into a file called `googleAnalytics.js` in the `src` directory. I named the file name specifically enough to make sure that the code is for that one purpose. I'm not sure how the possibility of people wanting to add other libraries into the file that I made called `googleAnalytics.js` applies here.